src/options.cpp

```diff
@@ -1035,6 +1018,57 @@ std::vector<options_manager::id_and_option> options_manager::build_soundpacks_li
     return result;
 }
 
+std::unordered_set<std::string> options_manager::get_lang_list()
```

Comment:
There ought to be a better name for this function.  Maybe something like `get_langs_with_translation_files`.